### PR TITLE
Rails react

### PR DESCRIPTION
**Pull request by mengiefen**
- In this project, the `rails-react` gem is used to connect Ruby on Rails and React front end. 
- The redux/toolkit is used as a state management system
- react-router-dom for routing in react front end also included.
- The project can show random 5 greetings on the static view page.

### CODE REVIEW

File: app/javascript/redux/greetingSlice.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+//   try {
+//     await fetch("http://127.0.0.1:3000/api/v1/greeting").then((response) =>
+//       dispatch(getGreeting(response.json()))
+//     );
+//   } catch (err) {
+//     throw new Error(err);
+//   }
+// };
+
+// export default greetingReducer;
```
Comment (t-yanick) [inline app/javascript/redux/greetingSlice.js]: - [x] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @mengiefen, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Your project is running as expected.

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/javascript/redux/greetingSlice.js (hunk 1 above)
Comment (mengiefen) [reply app/javascript/redux/greetingSlice.js]: Thank you!

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Mengstu, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**